javascript/cwp-pointers.js

```diff
@@ -0,0 +1,20 @@
+(function ($, cwpp){
+
+    $(document).ready( function() {
+
+        $("#cwp_product_affiliate_link").on("keyup", function(e){
+            if($(this).val().indexOf("amazon") != -1){
+                $("#cwp_product_affiliate_link_upsell").show();
```

Comment:
@selul dont show the pop up for this product or dont show the popup for this user?